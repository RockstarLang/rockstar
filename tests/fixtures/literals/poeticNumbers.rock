Tommy was a lovestruck ladykiller
say Tommy

Sweet Lucy was a dancer
say Sweet Lucy

A killer is on the loose
whisper a killer

My dreams were ice. A life unfulfilled; wakin' everybody up, taking booze and pills
shout My dreams

Sweet Lucy's a danger to society
say Sweet Lucy

my number is a 57 + true 43
say my number


(Non-ASCII letters are allowed)
I was ı
my heart is full of röck
shout my heart

(Non-numeric characters expect the first . are ignored)
my love was A-A-A-Alice!
say my love


