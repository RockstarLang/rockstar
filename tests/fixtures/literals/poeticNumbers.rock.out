100
16
235
3.1415926535
1627
14
424
8
